packages/installer.vm/installer.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>installer.vm</id>
-    <version>0.0.0.20231009</version>
+    <version>0.0.0.20231014</version>
```

Comment:
This version may need to be updated if other PR related to the installer is merged first